src/mobile/src/main/java/tupperdate/android/onboarding/Onboarding.kt

```diff
@@ -179,7 +138,7 @@ private fun Onboarding(
 private fun ViewPhoneInput(
     phone: String,
     setPhone: (String) -> Unit,
-    error: Error?,
+    error: Int?,
```

Comment:
Generally, you don't want to pass `Int` resources for stateless components, since you're not guaranteed that the composable won't be called by a caller which only has some `String` texts.